lib/pbr.h

```diff
@@ -83,6 +83,13 @@ struct pbr_filter {
  * the user criteria may directly point to a table too.
  */
 struct pbr_action {
+	/* VLAN */
+	uint8_t pcp;
+	uint16_t vlan_id;
+	uint16_t vlan_flags;
+
+	uint8_t queue_id;
```

Comment:
why uint8? (I thought linux supports identifiers up to  2^16-1

Reply:
Seems like (per `man tc`) Linux traffic-control queues are specified as MAJOR:MINOR, where each component is 16 bits. I'll bump this up to `uint32_t` and change the `...UNDEFINED...` constant to 0, to be in line with Linux, and add a constant to make the maximum queue (since that may be implementation-dependent) compile-time configurable.

@louberger, let me know if this seems right to you.